src/main/java/university/University.java

```diff
@@ -35,6 +39,7 @@ public String getAcronym() {
     }
 
     public int getSpotsLeft() {
+        assert spotsLeft > 0 : "Cannot remove a spot when none are left";

```

Comment:
The logic here may be wrong, perhaps try `>=0`? 😅 🥶

Reply:
oh goodness me! I placed that at the wrong function!